bin/potion

```diff
@@ -267,7 +272,19 @@ action = ARGV[0]
 query = ARGV[1]
 
 case action
+when 'new'
+  PotionCommandLine.new(ARGV[1], ARGV[2], ARGV[3])
+when 'g', 'generate'
+  PotionCommandLine.generate(ARGV[1], ARGV[2], ARGV[3])
 when 'create'
+  # TODO: remove in the future
+  # provided for backwards compatibility
+  options = ARGV[2..-1]
+  if options.first && options.first.include?('--')
+    puts "**NOTICE** potion create is deprecated please use potion new my_app\n"
```

Comment:
`puts "**NOTICE** potion create is deprecated; please use 'potion new my_app'\n"`